### PR TITLE
Handcrafted rpmspec 2899

### PR DESCRIPTION
**Pull request by kirushik**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@wstephenson is your reviewer :dancers: check https://trello.com/c/AFNeVycP/2899-cnt-switch-from-gem2rpm-to-plain-spec-file-fix-suseconnect-update 
@wstephenson: Please review this pull request using our guidelines: 
* test for acceptance criteria / functionality 
* check if the new code is covered with tests 
* check for unintended consequences 
* encourage usage of the boyscout rule 
* make sure the code is architected in the best way 
* check that no unnecessary technical debt got introduced 
* make sure that no unnecessary FIXMEs or TODOs got added

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/22775/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 5e695557f126d99b5ab312f47fd740b18010846e into 74d3d48562e4b287f46fac293310bcca3b4b0e70 )

File: package/SUSEConnect.8.ronn [hunk 1]
```diff
@@ -82,19 +96,25 @@ Manage subscriptions at [SUSE Customer Center][scc]
...
+  SUSEConnect can also be used to register systems with a local SUSE
+  Subscription Management Tool, instead of the SUSE Customer Center.
+  Use `SUSEConnect --url <smt-server-url>` to register systems with SMT and set
+  [proxies][ENVIRONMENT] as required.
 
   TBD
 
 ## IMPLEMENTATION
-  SUSEConnect is implemented in Ruby 2.0.  It communicates with the registration server using a RESTful JSON API over HTTP using TLS encryption.
+  SUSEConnect is implemented in Ruby 2.0.  It communicates with the registration
```
Comment (kalabiyau) [inline package/SUSEConnect.8.ronn]: I think we should drop part with 0 here as we not running on 2.0

File: package/SUSEConnect.changes [hunk 2]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Nov 19 15:36:56 UTC 2015 - kpimenov@suse.com
+
+- Update to version 0.2.30
+  - New packaging spec. One `SUSEConnect` package to tule them all
```
Comment (kalabiyau) [inline package/SUSEConnect.changes]: `tule -> rule` I guess

File: package/SUSEConnect.spec [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
```
Comment (kalabiyau) [inline package/SUSEConnect.spec]: that is wrong preamble, I think. 

http://bugs.opensuse.org/ -> https://bugzilla.suse.com

File: package/SUSEConnect.spec [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+#
+#
+# This file was generated with a gem2rpm.yml and not just plain gem2rpm.
```
Comment (kalabiyau) [inline package/SUSEConnect.spec]: wasn't that the whole point to not to use gem2rpm?

File: package/SUSEConnect.spec [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+done
+
+%build
+
+%install
+gem install --verbose --local --build-root=%{buildroot} \
+  --no-rdoc --no-ri \
+  %{mod_full_name}.gem
+
+
```
Comment (kalabiyau) [inline package/SUSEConnect.spec]: two lines rather than one, sorry, I am like that, yes :)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/22783/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged e67f7cde1ab747cfccbdd33f6f394895b2c4c3fe into 74d3d48562e4b287f46fac293310bcca3b4b0e70 )

File: package/SUSEConnect.spec [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+        echo "url: $reg_server" > /etc/SUSEConnect
+    fi
+fi
+
+# remove update-alternatives config for SUSEConnect
+if update-alternatives --config SUSEConnect  &> /dev/null ; then
+  update-alternatives --quiet --remove-all SUSEConnect
+  ln -s SUSEConnect.%{rb_default_ruby_suffix} %{_bindir}/SUSEConnect
+fi
+
```
Comment (kalabiyau) [inline package/SUSEConnect.spec]: I am again about the amount of empty lines

File: package/sles12.spec.erb [hunk 7]
```diff
@@ -1,215 +0,0 @@
...
-%files <%= custom_pkg_name %><%= config[:version_suffix] %>
-%defattr(-,root,root,-)
-<%= data[:filelist] %>
-
-<%   end
-   end
--%>
-%gem_packages
-
-%changelog
```
Comment (kalabiyau) [inline package/sles12.spec.erb]: :+1:

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/22788/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 6513eee15a0a0055069f0c400f318bbf362c3933 into 74d3d48562e4b287f46fac293310bcca3b4b0e70 )

**Pull request merged**